perf/perf_pmu.py

```diff
@@ -112,7 +112,8 @@ def test_config_PMU_sysfs(self):
         sysfs_dict = {"power8": ['mmcr0', 'mmcr1', 'mmcra'],
                       "power8e": ['mmcr0', 'mmcr1', 'mmcra'],
                       "power9": ['mmcr0', 'mmcr1', 'mmcra'],
-                      "power10": ['mmcr0', 'mmcr1', 'mmcr3', 'mmcra']}
+                      "power10": ['mmcr0', 'mmcr1', 'mmcr3', 'mmcra'],
+                      "power11": ['mmcr0', 'mmcr1', 'mmcr3', 'mmcra']}
```

Comment:
I think @disgoel we spoke last time as well, not sure how hard this to drive from YAML as code changes is always cost maintenance

Reply:
I agree with @PraveenPenguin . Better to pass from Yaml or use a config file.

Reply:
Have introduced a cfg file and introduced configparser to read the data from it.